test/integration/targets/ec2_vpc_route_table/tasks/main.yml

```diff
@@ -583,6 +583,19 @@
       filters:
         route-table-id: "{{ create_private_table.route_table.id }}"
       <<: *aws_connection_info
+    register: route_table_facts
```

Comment:
It would be good to name this var and its usages to `route_table_info`, to be consistent with the renaming of the module.